docs/we_made_this.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Who Made This
+description: Blog posts, tutorials, and guides about Lambda Powertools created by the AWS Community.
+---
+
+<!-- markdownlint-disable MD043 -->
+
+Lambda Powertools is a project created and maintained with a focus on helping the AWS community adopt best practices and Serverless. So this space is dedicated to convey the value of Powertools, use cases and lessons learned from the experience of AWS community.
+
+Do you write about Lambda Powertools or have an interesting use case to share? [Open an issue](https://github.com/awslabs/aws-lambda-powertools-python/issues/new?assignees=&labels=community-content&template=share_your_work.yml&title=%5BI+Made+This%5D%3A+%3CTITLE%3E) and we'll add your case here.
+
+## Contributors
+
+### Ran Isenberg
+
+Author info like twitter/preferred email... maybe a one line bio?
```

Comment:
@heitorlessa I'm not sure the best way to share the data like social network and email that authors send. Do you think it's a good idea to put it here more briefly?